site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -3047,7 +3047,7 @@ public function ajaxClearPeerMarks($gradeable_id) {
             $ta_graded_gradeable->deleteGradedComponent($component, $this->core->getQueries()->getUserById($peer_id));
         }
         $ta_graded_gradeable->removeOverallComment($peer_id);
-        $this->core->getQueries()->deleteOverallComment($gradeable_id, $peer_id, $gradeable->isTeamAssignment());
```

Comment:
What is the behavior change here? Why was the original code setup to remove from `gradeable_data` in addition to `gradeable_data_overall_comment`? I'm guessing we still want that removal from `gradeable_data` as we're clearing a peer grading here, so the data is no longer valid.